javascript/src/tiptap_gw/caption.tsx

```diff
@@ -65,13 +76,17 @@ function CaptionView(props: NodeViewProps) {
     return (
         <NodeViewWrapper className="richtext-evidence">
```

Comment:
Could we use a different class here or add a class to give the caption object different `padding` values than `richtext-evidence`? The `padding: 1rem` works well for the evidence object, but adds too much space above and below "Figure #" and the caption textbox.